app/src/main/java/io/homeassistant/companion/android/controls/ClimateControl.kt

```diff
@@ -60,22 +63,31 @@ object ClimateControl : HaControl {
             }
         val temperatureFormatSize = if (temperatureStepSize < 1f) "1" else "0"
         val rangeTemplate = RangeTemplate(
-            entity.entityId,
+            info.systemId,
             minValue,
             maxValue,
             currentValue,
             temperatureStepSize,
             "%.${temperatureFormatSize}f $temperatureUnit"
         )
         if (entityShouldBePresentedAsThermostat(entity)) {
+            val state = ClimateState(entity.state, ArrayList())
+            val toggleRangeTemplate = ToggleRangeTemplate(
+                info.systemId + "_range",
+                true,
```

Comment:
Could you add a comment at the end of this line explaining this is `true` so that the temperature will always be shown?

Reply:
Have added that. I've put it in a separate line as `ktlint` doesn't like comments in parameter lists at the end of the line